cnd/src/http_api/routes/rfc003/accept.rs

```diff
@@ -53,15 +51,31 @@ pub struct OnlyRefund<L: Ledger> {
     pub beta_ledger_refund_identity: L::Identity,
 }
 
-impl<B: ledger::Bitcoin + 'static> ListRequiredFields for Accept<B, Ethereum> {
+impl ListRequiredFields for Accept<bitcoin::Mainnet, Ethereum> {
```

Comment:
Does this need to be done now?

1. I am not sure I understand the point of this change.
1. If this is not needed for the extraction to a lib then we should avoid it to make this 168files PR easier to review.

Reply:
Unfortunately yes, same reason as here: https://github.com/comit-network/comit-rs/pull/1970#discussion_r374798588